core/src/apps/ethereum/networks.py.mako

```diff
@@ -58,7 +58,7 @@ def by_slip44(slip44: int) -> EthereumNetworkInfo:
 
 # fmt: off
 def _networks_iterator() -> Iterator[NetworkInfoTuple]:
-% for n in sorted(supported_on("trezor2", eth), key=lambda network: (int(network.chain_id), network.name)):
+% for n in sorted(supported_on("t2t1", eth), key=lambda network: (int(network.chain_id), network.name)):
```

Comment:
this is currently identical across models but we should do the same thing for T2B1 here as we do for coin_info

Reply:
Done in [1512420](https://github.com/trezor/trezor-firmware/pull/3240/commits/1512420557f66045b63527b6b3f0393024a5b889)